From 56164ab1f76ad2574dd27e19cdb9e390e704e13c Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Wed, 24 May 2017 17:01:09 +0800
Subject: [PATCH 1937/5242] MLK-15001-36 arm64: dtsi: fsl-imx8qxp: Set the 1st
 instance num for pd_mipi_dsi

commit  ad2a9d72956f04024154d18a189d9c4f092d53c1 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch sets the first instance number for pd_mipi_dsi.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |   14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index 9d094ea..dbdd411 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -605,30 +605,30 @@
 			};
 		};
 
-		pd_mipi_dsi: PD_MIPI_0_DSI {
+		pd_mipi_dsi0: PD_MIPI_0_DSI {
 			compatible = "nxp,imx8-pd";
 			reg = <SC_R_MIPI_0>;
 			#power-domain-cells = <0>;
 			#address-cells = <1>;
 			#size-cells = <0>;
 
-			pd_mipi_dsi_i2c0: PD_MIPI_0_DSI_I2C0 {
+			pd_mipi_dsi_0_i2c0: PD_MIPI_0_DSI_I2C0 {
 				name = "mipi0_dsi_i2c0";
 				reg = <SC_R_MIPI_0_I2C_0>;
 				#power-domain-cells = <0>;
-				power-domains =<&pd_mipi_dsi>;
+				power-domains =<&pd_mipi_dsi0>;
 			};
-			pd_mipi_dsi_i2c1: PD_MIPI_0_DSI_I2C1 {
+			pd_mipi_dsi_0_i2c1: PD_MIPI_0_DSI_I2C1 {
 				name = "mipi0_dsi_i2c1";
 				reg = <SC_R_MIPI_0_I2C_1>;
 				#power-domain-cells = <0>;
-				power-domains =<&pd_mipi_dsi>;
+				power-domains =<&pd_mipi_dsi0>;
 			};
-			pd_mipi_pwm0: PD_MIPI_0_DSI_PWM0 {
+			pd_mipi_0_pwm0: PD_MIPI_0_DSI_PWM0 {
 				name = "mipi0_dsi_pwm0";
 				reg = <SC_R_MIPI_0_PWM_0>;
 				#power-domain-cells = <0>;
-				power-domains =<&pd_mipi_dsi>;
+				power-domains =<&pd_mipi_dsi0>;
 			};
 		};
 
-- 
1.7.9.5

